runtime/doc/various.txt

```diff
@@ -416,8 +416,9 @@ m  *+xpm_w32*		Win32 GUI only: pixmap support |w32-xpm-support|
 			To stop the messages and commands from being echoed to
 			the screen, put the commands in a function and call it
 			with ":silent call Function()".
-			An alternative is to use the 'verbosefile' option,
-			this can be used in combination with ":redir".
+			An alternative is to use the 'verbosefile' option or
+			|:capture| function, this can be used in combination
```

Comment:
And “An alternative is {multiple alts}” is incorrect: this should be “The(?) alternatives are …”.